src/test/java/org/prebid/server/bidder/outbrain/OutbrainBidderTest.java

```diff
@@ -225,6 +229,55 @@ public void makeBidsShouldReturnResponseWithErrorWhenIdIsNotFound() throws JsonP
         assertThat(result.getValue()).isEmpty();
     }
 
+    @Test
+    public void makeBidsShouldReturnNativeBidWithModifiedImpIfNativeIsPresent() throws JsonProcessingException {
+        // given
+        final List<EventTracker> eventTrackers = new ArrayList<>();
+        final String impUrl = "impUrl";
+        final String jsUrl = "jsUrl";
+        final EventTracker impTracker = EventTracker.builder()
+                .event(1)
+                .method(1)
+                .url(impUrl)
+                .build();
+        final EventTracker jsTracker = EventTracker.builder()
+                .event(1)
+                .method(2)
+                .url(jsUrl)
+                .build();
+
+        eventTrackers.add(impTracker);
+        eventTrackers.add(jsTracker);
+
+        final Response response = Response.builder()
+                .eventtrackers(eventTrackers)
+                .build();
+
+        final String expectedAdm = jacksonMapper.encode(Response.builder()
+                .eventtrackers(null)
+                .jstracker(String.format("<script src=\"%s\"></script>", jsUrl))
+                .imptrackers(Collections.singletonList(impUrl))
```

Comment:
Let's avoid full Collections import and write it as `.imptrackers(singletonList(impUrl))`